yaml-tests/src/test/java/YamlIntegrationTests.java

```diff
@@ -258,4 +258,9 @@ public void enumTest(YamlTest.Runner runner) throws Exception {
     public void uuidTest(YamlTest.Runner runner) throws Exception {
         runner.runYamsql("uuid.yamsql");
     }
+
+    @TestTemplate
+    public void tableFunctionsTest(YamlTest.Runner runner) throws Exception {
```

Comment:
I am not really sure how this can pass with the plan serialization problem I think exists due to the `FirstOrDefaultStreamingValue` problem.

Reply:
what's the plan serialization problem?
the `FirstOrDefaultStreamingValue` is not even serialized, it is only used in the `DerivationProperty` to represent the semantics of the derivation.